### DISCUSSION TITLE
Sony世界攝影大獎冠軍作品 竟然是AI畫出來的！

### DISCUSSION THREAD

**Discussion by Miinteaa** [1101-GS4719-C/Python-Community-GS4719-C#1463] [General]
知名攝影比賽「Sony世界攝影大獎」（Sony World Photography Awards）日前公布2023年的獲獎名單，其中獲得創意類別冠軍的作品並非真實拍攝的照片，而是透過人工智慧（AI）系統生成的圖像。

德國攝影師艾達格森（Boris Eldagsen）的黑白肖像攝影作品「電工」（The Electrician）看起來是以傳統攝影技術所拍攝的、兩名來自不同世代的女性。
![19356DA6-4933-406A-B3F9-20CB6A3716A2](https://github.com/NCU-GS4719-Python/Python-Community/assets/125457513/fe98cce8-b6c9-4a59-b2f8-eb3d74d0f89b)

不過，艾達格森於獲獎當天在FB發文[坦承](https://www.facebook.com/boriseldagsen/posts/6115217061926857)，這張其實是由Open AI的「DALL·E 2」繪圖程式生成的圖像，並拒絕接受Sony世界攝影大獎提供的相機獎品與5000美元（約新台幣15萬元）獎金，他認為，AI產出的圖像與攝影不相同，不該放在同個比賽中競爭，「因此我不會接受這個獎項」。

事實上，「電工」是艾達格森2022年開始創作的「Pseudomnesia」系列作品之一；Pseudomnesia在拉丁語中，指的是「虛假記憶」，包括從未發生過的虛假事件，而不只是不準確的記憶。

艾達格森指出，他這麼做是想證明攝影比AI更有價值，而他獲獎這件事也證明針對AI圖像在業界的使用，需要更多討論並訂定規範。

https://www.upmedia.mg/news_info.php?Type=3&SerialNo=170604

**DALL•E 2介紹**
OpenAI 於2021年初所發佈的 DALL•E，是基於 GPT-3模型所開發出的AI繪圖模型，GPT-3是一種極其先進的語言生成模型，同樣也是由 OpenAI 所開發，擁有高達1750億個參數，其目的是讓電腦也能夠理解人類語言的溝通方式。
OpenAI 再結合另一個能夠辨識殘缺圖像並進行補全的「Image GPT」模型，進一步發展出能夠統合語言模型與圖像模型的AI神經網路 DALL•E。
DALL•E的最大特點，即是可以依據使用者輸入的自然語意的文本內容，生成合乎指定主題的圖像，並且可以指定圖片 _風格類型_，而不是單純地將不同物件進行縫合。

**個人感想：**
其實在AI繪圖技術出現後已有多例使用AI繪製畫作參加競賽並得名的案例，但用來參加「攝影比賽」還是第一次看到，甚至是世界級的競賽評審都無法分辨並排除AI運算出的「非真實」的圖片，令我非常吃驚。
身為一個有在接觸電腦繪圖的（算是？）創作者，我認為AI繪圖帶給社會的便利及成效是讓人驚艷的，但也的確的抹殺了身為創作者的生存空間以及創意、個人風格。
在科技發展飛快的時代裡，關於技術運用的規範及範圍可能也需要一併列入思考以及討論，才能在人和機之間保持平衡。

**Comment by Tzuhsuannnnn** [comment]
我認為攝影的目的主要是記錄某個時刻一瞬間的畫面，如果是用AI直接去生成就有點喪失攝影的目的了，但連評審都看不出來，感覺這真的會是個以後需要被關注的議題。

**Comment by sumairu0727** [comment]
最近AI繪圖的正確度越來越高，連半年前那種出圖出到手指歪掉得情況都越來越少見，常常很多人在討論會師會不會因此失去工作。
我認為確實應該限制給AI訓練圖片應該是由作者同意的才能放進訓練的資料庫，不然就算即使該AI沒有商業用途也足以對繪師造成巨大傷害。
但我認為繪師也有一些新的路可以嘗試，例如去修改AI生成圖的瑕疵，或者幫忙構圖再由AI工具輔助中間的繪圖步驟，或許可以走出不一樣的藝術未來

**Comment by brian431** [comment]
可能以後的攝影比賽必須採用某種標準讓參賽者可以證明攝影作品為自己所拍攝的，不過具體作法為何我就不知道了。

**Comment by juliaouo** [comment]
我認為攝影是表達拍攝者內心的情感投射，並在現實世界中找尋那一抹最符合的光景，包括構圖、色彩、焦距等都能體現拍攝者的情感。若是用 AI 生成的話不一定能找到最符合內心所想的畫面。另外攝影者會為了拍攝遊覽各地，我想這其中的過程才是最美好的所在。

**Comment by Aaron0810** [comment]
AI繪圖真的越來越先進，現在網路上這種AI生成的不論是繪圖或實際照片都越來越難辨識，很多作品就算給真的評論家來評論可能也判斷不出來，感覺未來在任何比賽或網路上應該要有個規範規定如果是AI生成的要明確標示出來才比較好。

**Comment by pudding0803** [comment]
現在有越來越多創作，包括繪圖、影像、文字等，都在逐漸讓 AI 加入。但除了會產生著作權的議題之外，過多的 AI 作品可能也會導致傳統作品的價值降低，畢竟接下來肯定是會越來越難辨別，產量過大難免會供過於求。但加以規範似乎又不太可行，希望未來能達到更好的平衡。

**Comment by Superlan8** [comment]
我覺得AI技術已經越來越先進，之後勢必要有法律來規範合理用途，但這些又涉及到著作權等問題，我想現在可能要開始思考，就像歐盟的《AI責任指令草案》，雖然可能不是到非常完善，但至少要開始重視AI的法律問題了。

**Comment by youtinglin** [comment]
感覺真的要更完善訂定一些規則或法規，不然感覺競賽會沒有實感，因為攝影就是要體會拍攝當下的光線和角度，甚至是氛圍感和視覺角度及傳達的意義，才是攝影比賽的意義，如果用AI模型生成，就沒有辦法體會到在快門按下後成果展現的感受。

**Comment by zhiron544** [comment]
我覺得拍照的意義原先就是要講眼前的事物記錄下來，因此用AI繪圖去獲得比賽，從最根本的意義就有所偏差，現實是解中既不存在那個場景，也沒有記錄下畫面的那個瞬間。
